tests/queries/0_stateless/02250_ON_CLUSTER_grant.reference

```diff
@@ -0,0 +1,4 @@
+with_on_cluster_02250_ON_CLUSTER_grant_default
+localhost	9000	0		0	0
```

Comment:
Here is where the test failed, see https://s3.amazonaws.com/clickhouse-test-reports/35767/5920a535ff75e2484260a76807d3e496ea1508b7/stateless_tests__release__databasereplicated__actions__[1/2].html

I think we can redirect the output of the command `CREATE DATABASE` to `/dev/null` and only check stderr

Reply:
>I think we can redirect the output of the command CREATE DATABASE ON CLUSTER to /dev/null and only check stderr

Yep, for this test it is not important.